bio_files_processor.py

```diff
@@ -0,0 +1,106 @@
+import os
+from Nika_modules import parser_gbk
+
+
+def convert_multiline_fasta_to_oneline(input_fasta, output_fasta=None):
+    """
+    This function takes multiline fasta file as an input and convert the multiline sequences in one line.
+    :param input_fasta: str. path to multiline fasta file
+    :param output_fasta: str. name of new one line fasta file. This parameter isn't necessary. 
+    :return: file. 
+    """
+    try:
+        with open(input_fasta, 'r') as input_file:
+            fasta_data = input_file.read()
+    except FileNotFoundError:
+        return "Input fasta file not found."
+
+    oneline_fasta = []
+    fasta_parts = fasta_data.split('\n>')
+
+    for part in fasta_parts:
+        if part:
+            lines = part.strip().split('\n')
+            header = lines[0]
+            sequence = ''.join(lines[1:])
+            oneline_fasta.append(f'>{header}\n{sequence}')
+
+    oneline_fasta = '\n'.join(oneline_fasta)
+
+    if not os.path.exists("/Users/veronikavadehina/Desktop/Bioinf "
+                          "23:24/Python_Nika_GitHub/bioinf_for_py/fasta_oneline_rewrite_files"):
+        os.makedirs("/Users/veronikavadehina/Desktop/Bioinf "
+                    "23:24/Python_Nika_GitHub/bioinf_for_py/fasta_oneline_rewrite_files")
```

Comment:
😳😳😳
Я прогнал, не знаю, создалась ли там у тебя папка:)))

Reply:
Если создалась получается всё работает!

Reply:
```suggestion
    if not os.path.isdir("fasta_oneline_rewrite_files"):
        os.makedirs("fasta_oneline_rewrite_files")
```

Не надо прописывать в общем абсолютные пути, как минимум в качестве дефолтных. Как писал мне мой преподаватель по питону:
![photo_2022-12-14_11-09-11](https://github.com/nikavvv57/bioinf_try2/assets/76266535/627c54c2-95eb-405c-8add-7698471b9e30)